plugins/huggingface/api_token.go

```diff
@@ -0,0 +1,88 @@
+package huggingface
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIToken,
+		DocsURL:       sdk.URL("https://huggingface.co/docs/huggingface_hub/quick-start"), 
+		ManagementURL: sdk.URL("https://huggingface.co/settings/tokens"), 
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Token,
```

Comment:
I'd propose updating the field name to "User Access Token" too if we update the credential, because that's the name Hugging Face is using throughout the [environment variables documentation](https://huggingface.co/docs/huggingface_hub/package_reference/environment_variables#huggingfacehubtoken) as well.